.gitignore

```diff
@@ -5,3 +5,5 @@ target
 build/config.mak
 build/Makefile.version
 test/config
+build/x64
+build/x86
```

Comment:
This is actually wrong. Please add a "/" before every line in this file (except for the comment and blank line).

If you have "build/x86", then that ignores "build/x86" in any directory path, and that's not what you want.

If you have "/build/x86", that that ignores build/x86 from the root of the .gitignore file, which is what you want.  The prior lines were in error too, we learned this after setting up OMS-Agent-For-Linux.

Thanks.